build.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+VER=$(sed -n 's/\PACKAGE_VERSION="\(.*\)"/\1/p' dkms.conf)
+sudo rsync -rvhP ./ /usr/src/rtl88x2bu-${VER}
+sudo dkms add -m rtl88x2bu -v ${VER}
```

Comment:
DRV_NAME=rtl88x2bu
dkms add -m ${DRV_NAME} -v ${VER}
dkms build -m ${DRV_NAME} -v ${VER}
dkms install -m ${DRV_NAME} -v ${VER}

Reply:
Updated the source file. Thanks!!